### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by gm7**
Removed unnecessary files

### CODE REVIEW

File: teamfu/.gitignore [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+# User-specific files
+*.suo
+*.user
+*.userosscache
+*.sln.docstates
```
Comment (TillCM) [inline teamfu/.gitignore]: Can we keep this file for visual studio?**please**

**Review by TillCM — CHANGES_REQUESTED** [review]
Please comment  in the changes you made in the code for documentation

File: teamfu/.gitignore (hunk 1 above)
Comment (gm7) [reply teamfu/.gitignore]: @TillCM addressed in https://github.com/TillCM/teamfu/pull/3/commits/3fdefcc6f2ed85d01ee2efd5b2a03f765aa8c35a

**Review by TillCM — APPROVED** [review]
LGTM

**Pull request merged**